app/controllers/works_controller.rb

```diff
@@ -0,0 +1,95 @@
+class WorksController < ApplicationController
+  # Note: no before action because the example site lets us view, delete, edit, and create works without logging in
+
+  def index
+    @movies = Work.sort_movies
+    # TODO - not working
+    # @movies = Work.sort_type("movie")
+    @books = Work.sort_books
+    @albums = Work.sort_albums
+  end
+
+  def home
+    @movies = Work.sort_movies[0..9]
+    @books = Work.sort_books[0..9]
+    @albums = Work.sort_albums[0..9]
+    @spotlight = Work.spotlight
+  end
+
+  def show
+    @work = Work.find_by(id: params[:id])
+    if @work.nil?
+      head :not_found
+      return
+    end
```

Comment:
You do this in a number of places.  In the future you should DRY things like this up by creating a controller filter.